core/src/test/scala/com/nvidia/spark/rapids/tool/tuning/ProfilingAutoTunerSuite.scala

```diff
@@ -3371,18 +3335,16 @@ class ProfilingAutoTunerSuite extends BaseAutoTunerSuite {
           |--conf spark.rapids.shuffle.multiThreaded.reader.threads=28
           |--conf spark.rapids.shuffle.multiThreaded.writer.threads=28
           |--conf spark.rapids.sql.batchSizeBytes=2147483647b
-          |--conf spark.rapids.sql.concurrentGpuTasks=2
+          |--conf spark.rapids.sql.concurrentGpuTasks=3
           |--conf spark.rapids.sql.format.parquet.multithreaded.combine.waitTime=1000
           |--conf spark.rapids.sql.multiThreadedRead.numThreads=80
           |--conf spark.rapids.sql.reader.multithreaded.combine.sizeBytes=10m
           |--conf spark.shuffle.manager=com.nvidia.spark.rapids.spark$testSmVersion.RapidsShuffleManager
-          |--conf spark.sql.adaptive.advisoryPartitionSizeInBytes=32m
           |--conf spark.sql.adaptive.autoBroadcastJoinThreshold=[FILL_IN_VALUE]
-          |--conf spark.sql.adaptive.coalescePartitions.initialPartitionNum=800
           |--conf spark.sql.adaptive.coalescePartitions.minPartitionSize=4m
           |--conf spark.sql.adaptive.coalescePartitions.parallelismFirst=false
           |--conf spark.sql.files.maxPartitionBytes=512m
-          |--conf spark.sql.shuffle.partitions=800
+          |--conf spark.sql.shuffle.partitions=100
```

Comment:
QQ: why would the autotuner result change for this test case (and for the previous test cases too)?

Reply:
That is a good question. We do not have benchmark for AQE and related shuffle partition L4. Created a parent issue to benchmark tuning configs for L4 - https://github.com/NVIDIA/spark-rapids-tools/issues/1682.

Updated such tests for `ignore()` for now.

cc: @amahussein

Reply:
scary!

Reply:
The UT changes in this PR are a due to updating the default instance types used by CSPs to match those in the Qualification model. 
Ideally, I could have made these changes in a separate PR which might have reduced the impact on the UTs.